contracts/src/v0.8/interfaces/VRFCoordinatorV2Interface.sol

```diff
@@ -113,4 +113,14 @@ interface VRFCoordinatorV2Interface {
    * @param to - Where to send the remaining LINK to
    */
   function cancelSubscription(uint64 subId, address to) external;
+
+  /*
+   * @notice Check to see if there exists a request commitment consumers
+   * for all consumers and keyhashes for a given sub.
+   * @param subId where to send the funds
```

Comment:
This param doc doesn't make much sense in the context of this method name (since there's no fund sending involved here). Maybe `@param subId the subscription ID to check for pending requests`? A bit redundant but better than mentioning funding IMO.

Reply:
Hmm yeah, this was copied from the coordinator but looks wrong. Updated to just "subId - ID of the subscription" to match the above funcs